ui/app/pages/unlock-page/unlock-page.component.js

```diff
@@ -35,7 +35,7 @@ export default class UnlockPage extends Component {
   UNSAFE_componentWillMount () {
     const { isUnlocked, history } = this.props
 
-    if (isUnlocked) {
+    if (isUnlocked && history) {
```

Comment:
Why is this ever undefined? Can we make it `.isRequired` and make sure it's passed in everywhere? Or am I missing something?

Reply:
that's a very good point, I can add that work to this PR